### PR TITLE
Cannot list payments due to geolocation. (Bunq/sdk_java#46 )

### PR DESCRIPTION
**Pull request by OGKevin**
Closes bunq/sdk_java#46
- [x] Tested

### CODE REVIEW

**Comment by OGKevin** [comment]
@patrickdw1991 all yours please 👀

File: src/main/java/com/bunq/sdk/json/BigDecimalTypeAdapter.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public void write(JsonWriter output, BigDecimal value) throws IOException {
 
   @Override
   public BigDecimal read(JsonReader input) throws IOException {
-    if (input.peek() == JsonToken.STRING) {
+    if (input.peek() == JsonToken.NUMBER) {
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/json/BigDecimalTypeAdapter.java]: Why not just make this handle both strings and numbers?
```
if (type = number) {
    new bigdecimal(getDoubleValue)
} else if (type = string) {
    new bigdecimal(getStringValue)
} else {
    null
}
```

File: src/main/java/com/bunq/sdk/json/BigDecimalTypeAdapter.java [hunk 2]
```diff
@@ -29,7 +29,11 @@ public void write(JsonWriter output, BigDecimal value) throws IOException {
 
   @Override
   public BigDecimal read(JsonReader input) throws IOException {
-    if (input.peek() == JsonToken.NUMBER) {
+    JsonToken type = input.peek();
+
+    if (type == JsonToken.NUMBER) {
+      return new BigDecimal(input.nextInt());
```
Comment (patrickdw1991) [inline src/main/java/com/bunq/sdk/json/BigDecimalTypeAdapter.java]: Why nextInt instead of double? Int will not return you the fractions.

Comment (OGKevin) [reply src/main/java/com/bunq/sdk/json/BigDecimalTypeAdapter.java]: 😂 🤦‍♂️

**Comment by patrickdw1991** [comment]
@andrederoos

**Comment by OGKevin** [comment]
@andrederoos merging!

**Pull request merged**